browser/ui/webui/playlist_page_handler.cc

```diff
@@ -72,6 +76,20 @@ void PlaylistPageHandler::AddMediaFilesFromPageToPlaylist(const std::string& id,
                                                                   url.spec());
 }
 
+void PlaylistPageHandler::AddMediaFilesFromOpenTabsToPlaylist(
```

Comment:
I'm not sure I understand what's happening here, we're adding all media files from open tabs except this tab?

Reply:
oh I see, web_contents_ is the webui webcontents. This is not correct then because the webui webcontents is not associated with a browser

Reply:
> webui webcontents is not associated with a browser

Sorry, I'm not sure what you mean here. Currently, `brave://playlist` is loaded on a tab and I thought we should filter it from contents of tab strip model.